### PR TITLE
Modified Home Page as per suggestions provided, Added Sponsorship Page

### PR DESCRIPTION
**Pull request by rmaheshkumarblr**
Updated screenshot in Slack Group

### CODE REVIEW

**Comment by rmaheshkumarblr** [comment]
image

File: sponsorship.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+layout: page
+title: Sponsorship
+permalink: /sponsorship/
+---
+
+The Colorado Data Science Team hosts several competitions each semester with prizes for the top three teams. 
+<ul>
+ <li> These prizes are often sponsored by a local business or nonprofit organization, and in exchange, we invite the sponsor to come to a team meeting at the beginning or end of the competition to tell us about their organization and how they use, or would like to use, data science or machine learning techniques to advance their goals.</li>
```
Comment (EntilZha) [inline sponsorship.md]: Lets use markdown here instead of html list elements

Comment (rmaheshkumarblr) [reply sponsorship.md]: Any particular reason why markdown is preferred over html list elements?

Comment (EntilZha) [reply sponsorship.md]: For markdown files should stuck to using markdown since thats sort of the point.

**Comment by EntilZha** [comment]
Minor thing, if you click "about" then hover over the text it turns white with a white background. Perhaps override the css color on hover to not change to white text.

**Comment by rmaheshkumarblr** [comment]
Updated the CSS to ignore color change on hover

**Comment by rmaheshkumarblr** [comment]
image
- [x] Missing the title/header element of “Recent News” above the postings.
- [x] Images should scale correctly on the carousel (not overflow). In general the element could be less tall to line up better with the other elements in the row 
- [x] The gray/white gradient seems out of place and pops out too much, would just remove that CSS and leave it white
- [x] Add a title/header above the carousel saying something like “Competitions” so that its clear thats what it is sliding through

**Comment by EntilZha** [comment]
LGTM, thanks for the great work

**Pull request merged**